analyzers/tests/SonarAnalyzer.UnitTest/Rules/TestMethodShouldContainAssertionTest.cs

```diff
@@ -55,28 +56,30 @@ private static class XUnitVersions
             public const string Ver2 = "2.0.0";
         }
 
+        private readonly VerifierBuilder builder = new VerifierBuilder<TestMethodShouldContainAssertion>();
+
         [DataTestMethod]
         [DataRow(MsTestVersions.Ver1)]
         [DataRow(Constants.NuGetLatestVersion)]
         public void TestMethodShouldContainAssertion_MSTest(string testFwkVersion) =>
-            OldVerifier.VerifyAnalyzer(new[] { @"TestCases\TestMethodShouldContainAssertion.MsTest.cs", @"TestCases\TestMethodShouldContainAssertion.MsTest.AnotherFile.cs" },
-                new TestMethodShouldContainAssertion(),
-                AdditionalTestReferences(NuGetMetadataReference.MSTestTestFramework(testFwkVersion)));
+            builder.AddPaths(new[] { "TestMethodShouldContainAssertion.MsTest.cs", "TestMethodShouldContainAssertion.MsTest.AnotherFile.cs" })
```

Comment:
It's not necessary to create a new array here since the parameter is defined with `params`.